code/game/objects/items/weapons/shields.dm

```diff
@@ -70,12 +70,20 @@
 	var/active = 0
 
 /obj/item/shield/energy/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
+	if(istype(hitby, /obj/item/projectile))
+		var/obj/item/projectile/P = hitby
+		if(P.shield_buster && active)
+			toggle(owner)
```

Comment:
This can make a clown hit himself. As well as adds a fingerprint to the shield. Might want to recheck this

Reply:
Fixed